FogCreek/Glitch-Community
# Pool sweatpants: resend invites for team users, delete invited user

## Links
* Remix link: https://pool-sweatpants.glitch.me/
* Manuscript link:

 related to https://glitch.manuscript.com/f/cases/3326592/teams-persistently-see-avatars-of-people-that-have-been-invited-to-a-team
* Specs/Notion docs: https://www.notion.so/glitch/87bda09f85184c1d81e1659aa62e4392?v=9f7963becc9341ed9af502682c6e9eef&p=7de6515edd654496af164018eb597c0a

## GIF/Screenshots:
![Screen Shot 2019-06-07 at 2 55 37 PM](https://user-images.githubusercontent.com/3011231/59126940-53dfee80-8934-11e9-9d04-3379ecf3be66.png)
## Changes:
* Clicking an invited user will now get you a popup (previously went to their profile)
* That popup has link to profile, button to resend invite email, button to remove user from team

## How To Test:
* Add members to your favorite team, try the buttons to resend invites and revoke access
* Note that 429 rate limiting is still in effect for the emails

## Feedback I'm looking for:
* Does it work?
* Did I component it right? (if so, all accolades should go to Tara)
* Is anything missing?

## Things left to do before deploying:
- have a dance party? IDK...
